src/main/component/Editor.svelte

```diff
@@ -82,24 +87,36 @@
   let editorRef: TinyMCEEditor | null;
   let lastVal = value;
   let disablindCache = disabled;
+  let readonlyCache = readonly;
   
+  const setReadonly = (editor: TinyMCEEditor, readonlyValue: boolean) => {
+    if (typeof editor.mode?.set === 'function' && isDisabledOptionSupported(editor)) {
```

Comment:
```suggestion
    if (typeof editor.mode?.set === 'function') {
```
readonly should work for older versions of TinyMCE as well.

Reply:
My thought is if I use this wrapper with an older TinyMCE (< 7.4), I should be toggle the editor's readonly mode via `readonly` property

Reply:
@lorenzo-pomili, I am going ahead to accept this suggestion to fix the readonly prop not working with TinyMCE < 7.6